src/WaitingRoom.js

```diff
@@ -49,16 +73,21 @@ export default class WaitingRoom extends Component {
       )
     }
 
-    var cards = this.state.sessions.map((session) => (
+    var sessions = this.state.sessions.filter(this.getFilterFunction());
+
+    var cards = sessions.map((session) => (
       <GameCard
           key={session.id}
           style={cardStyle}
           session={session} />
     ));
 
     return (
-      <div>
-        <AddGameButton style={cardStyle} />
+      <div >
+        <div style={{display:'flex', justifyContent: 'space-between', margin: '20px'}}>
```

Comment:
Left this inline just to try it out. Let me know if you think the positioning makes sense and if I should extract the style object in a variable above.

Reply:
LGTM!